src/parser.y

```diff
@@ -172,7 +172,7 @@ static jv check_object_key(block k) {
     char errbuf[15];
     return jv_string_fmt("Cannot use %s (%s) as object key",
         jv_kind_name(block_const_kind(k)),
-        jv_dump_string_trunc(jv_copy(block_const(k)), errbuf, sizeof(errbuf)));
+        jv_dump_string_trunc(block_const(k), errbuf, sizeof(errbuf)));
```

Comment:
Was this a leak?

Reply:
There were plenty of leaks I've traced down and killed.
Apparently, this was one of them. To tell the truth I don't recall already but I can review. There were plenty of unobvious places which lead to my API renaming suggestion which hasn't caught on with the folks.

Reply:
No, it's ok, thanks.